static/app/views/dashboards/exportDashboard.tsx

```diff
@@ -0,0 +1,86 @@
+const delete_properties = ['createdBy', 'dateCreated', 'id', 'dashboardId', 'widgetId'];
+
+async function exportDashboard() {
+  const params = getAPIParams();
+  const api_url = `https://${params.base_url}/api/0/organizations/testorg-az/dashboards/${params.dashboard_id}/`;
+  const response = await fetch(api_url);
+  const jsonData = await response.json();
+  const normalized = normalize_data(jsonData);
+  normalized.projects = [];
+  downloadObjectAsJson(normalized, 'dashboard');
```

Comment:
@sergiosentry can we use the title of the dashboard as the filename instead of `'dashboard'`?

Reply:
I.e. if title is `uploaded - Web + Mobile (crashes + slowdowns)`

then strip out any characters that aren't letters or numbers, and tack on a timestamp to know when it was downloaded.

Filename could become `uploaded-web-mobile-crashes-slowdowns-07-03-2023.13.59.50.json`

Maybe overkill but would make it clearer which downloaded dashboard we're looking at?

You

Reply:
Yes, I agree - this sounds better